CONTRIBUTING.md

```diff
@@ -17,10 +17,17 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 ## <a name="question"></a> Got a Question or Problem?
 
-If you have questions about how to *use* Angular Material, please direct them to the 
-[Google Group][material-group] discussion list or [StackOverflow][stackoverflow]. 
-Please note that Angular Material is still in very early development, and the team's capacity 
-to answer usage questions is limited. Community chat is also available on [Gitter][gitter].
+Please, do not open issues for the general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](https://stackoverflow.com/questions/tagged/angular-material2) where the questions should be tagged with tag `angular-material2`.
```

Comment:
`Please, do not` -> `Please do not`

`for the general support questions` -> `for general support questions`